### PR TITLE
API provides functionality for database to store Inquiries 

### PR DESCRIPTION
**Pull request by LucasKnudsen**
[PT Story](https://www.pivotaltracker.com/story/show/178351102)

## User story
```
As a broker
In order to provide suggestions to potential customers 
I want to be able to store new inquiries 
```

## Changes proposed: 
- Adds Inquiry model
- Adds POST endpoint and create action functionality

### CODE REVIEW

File: spec/factories/inquiries.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :inquiry do
+    size { 1 }
+    variants { true }
```
Comment (lukeperrin10) [inline spec/factories/inquiries.rb]: What is variants doing here?

**Review by lukeperrin10 — COMMENTED** [review]
Just the question about the variants and what it's doing :)

File: spec/factories/inquiries.rb (hunk 1 above)
Comment (christerforsgren91) [reply spec/factories/inquiries.rb]: Its if they want open space or own room :)

**Review by lukeperrin10 — APPROVED** [review]

File: spec/factories/inquiries.rb (hunk 1 above)
Comment (offcast-dev) [reply spec/factories/inquiries.rb]: yes, it is not understandable. Can you do office true/false? This naming will cause confusion if future

**Review by offcast-dev — COMMENTED** [review]
Bette r to change naming

**Review by arunbhalli — APPROVED** [review]

**Pull request merged**